docs/development.rst

```diff
@@ -445,6 +448,86 @@ finally force-push to the remote topic branch on your fork::
 Hard resetting and force pushing are not reversible operations, so please
 beware!
 
+******************************
+Adding a new demographic model
+******************************
+
+Before implementing any model, be sure to have forked the `stdpopsim` repository
+and cloned it locaaly, following the instructions in the `GitHub Workflow`_ section.
+Models are first implemented and tested locally, and then submitted as a pull request
+to the `stdpopsim` repository, at which point it is verified by another developer
+before being fully supported within `stdpopsim`.
+
+Every demographic model has a few necessary features or attributes. First of all,
+demographic models are defined by the population sizes, migration rates, split and
+admixture times, and generation lengths given in the source publication. We often take
+the point estimates for each of the values from the best fit model (for example, the 
+parameters that give the maximum likelihood fit), which are translated into
+`msprime`-formatted demographic inputs.
+
+`Msprime`-defined demographic models are specified through the
+``population_configurations``, ``migration_matrix``, and ``demographic_events``. If this
+is your first time specifying a model using `msprime`, it's worth taking some time to
+read through the `msprime` documentation and tutorials
```

Comment:
Nicer to use a link here, like 
```
through the `msprime documentation <https://msprime.readthedocs.io/en/stable/tutorial.html>`_
```

or whatever the syntax is - I can never remember!